astropy/stats/sigma_clipping.py

```diff
@@ -6,6 +6,14 @@
 import warnings
 from ..utils.exceptions import AstropyDeprecationWarning, AstropyUserWarning
 
+try:
+    from scipy.stats import sigmaclip
+    HAS_SCIPY=True
+except:
+    warnings.warn("Scipy is required for some functionality of",
+            "sigma_clip. The `masked` keyword will be forced to `True`.")
```

Comment:
You may add ``AstropyUserWarning`` as the warning class, thus the user would know where it's coming from